GameOfLife/Calculations/Rules.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    class Rules
+    {
+        public int[,] CellCalculation(int[,] GameField)
+        {
+            int[,] tempArr = (int[,])GameField.Clone();
```

Comment:
tempArr - same comment, dont shorten words. Also, could improve on naming. What does the array represent? Its gonna be either current/previous/next generation of cells

Reply:
Renamed to nextGenerationOfCells since in this array is being saved next iteration of cells